stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -330,6 +334,11 @@ spec:
       - name: binarystore-xml
         configMap:
           name: {{ template "artifactory-ha.fullname" . }}-bs
+      {{- if .Values.artifactory.userPluginsSecret }}
+      - name : user-plugins
+        secret:
+          secretName: {{ .Values.artifactory.userPluginsSecret }}
```

Comment:
Would you be amenable to passing this through `tpl`? For our use case, we actually use Artifactory as a sub chart dependency of a 'wrapper' chart. This means that the name of the secret is actually dynamic based on the name of the Helm release.

Reply:
The same goes for all of the other StatefulSets as well obviously

Reply:
Will take a look

Reply:
@robbie-demuth We are not creating secret as part of helm deployment. I think you should be able to pass secret names using `--set` or through `values.yaml` file during helm installation.

Reply:
For our use case, that is not possible. We deploy a chart that marks this chart as a dependency, so the secret _is_ created in the same Helm release as StatefulSets for Artifactory. The name of the secret is also based on the name of the Helm release since it is something like `name: {{ template "artifactory-ha-wrapper.fullname" . }}`. It would be great if this line could call `tpl` so that we can pass `{{ template "artifactory-ha-wrapper.fullname" . }}` as a value and have the inner Artifactory chart resolve the templating language

Reply:
CC @valdisrigdon

Reply:
@robbie-demuth - I suggest you submit a PR with support for your method once this is merged. We welcome features that will help our charts users.